cmd/examples/nse/nse.go

```diff
@@ -32,6 +25,11 @@ import (
 	"github.com/ligato/networkservicemesh/plugins/nsmserver"
 	"github.com/sirupsen/logrus"
 	"google.golang.org/grpc"
+	"net"
```

Comment:
Really weird formatting, usually standard packages goes first, have you run gofmt?

Reply:
Yea, unfortunately gofmt doesn't catch these, it only ensures that the lists are sorted and we're responsible for separating them out. I'll fix this.